### PR TITLE
#577 speech bubble comment

### LINKED ISSUE #577: The comment column for supplier requisition lines should be the speech bubble column
Currently the comment column for supplier requisition lines is just a standard text cell.

Since there are so many cells, having the smaller speech bubble column would be good.

Should the comment be editable?

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #577 

Changed to the speech bubble comment style
Screen Shot 2022-04-08 at 1 58 31 PM

### CODE REVIEW

File: packages/host/src/CommandK.tsx [hunk 1]
```diff
@@ -166,13 +166,6 @@ export const CommandK: FC = ({ children }) => {
             .build()
         ),
     },
-    {
```
Comment (mark-prins) [inline packages/host/src/CommandK.tsx:169]: offroad: found that while I was testing on the demo server I was accidentally mashing the `r` key and getting a reports placeholder page 🙄

File: packages/host/src/Site.tsx [hunk 2]
```diff
@@ -40,11 +40,12 @@ const Heading: FC = ({ children }) => (
 
 const NotifyOnLogin = () => {
   const { success } = useNotification();
-  const { store } = useAuthContext();
+  const { store, storeId } = useAuthContext();
```
Comment (mark-prins) [inline packages/host/src/Site.tsx:43]: offroad: when testing the logged out alert, I noticed that when the `refreshToken` happened, the user received two toasts to say 'welcome to xx store' even though they have already been notified. Changing the reference here removes the duplication and removes the notification-on-refresh-token, which should not have been happening

**Review by CarlosNZ — COMMENTED** [review]
Cool, do you think the ResponseRequisition detail view should also use the popover comment column too, since it's basically the same pattern?

Even the Outbound/Inbound shipment lists too, perhaps?

**Comment by mark-prins** [comment]
> do you think the ResponseRequisition detail view should also use the popover comment column too, since it's basically the same pattern?

cool - have done. 

I've left Inbound / Outbound lists for now, since there's plenty of room and we have the 'colour' column on the LH end already

**Review by CarlosNZ — APPROVED** [review]
LGTM 👍

**Pull request merged**